load_game.py

```diff
@@ -68,25 +67,25 @@ def load_into_game(gamestate: str, step_counter: str):
         for key in results[gamestate][step_counter]:
             save_num = int(results[gamestate][step_counter][key]["save_num"])
 
-            nemesis = key
             if save_num > 200:
                 pass
+            elif key != str(game_vars.nemesis()):
+                pass
```

Comment:
Since this isn't doing anything other than controlling the else, it would likely be better to figure out when we want to meet into the else and specifically check that.